article/app/views/package.scala

```diff
@@ -29,7 +29,7 @@ object MainCleaner {
         if (amp) AmpEmbedCleaner(article) else VideoEmbedCleaner(article),
         PictureCleaner(article, amp),
         MainFigCaptionCleaner,
-        AtomsCleaner(article.content.atoms, shouldFence = true, amp)
+        AtomsCleaner(article.content.atoms, shouldFence = true, amp, mainMedia = true)
```

Comment:
Can we either have named parameters everywhere or no at all? You can put one param per line